Charter.md

```diff
@@ -0,0 +1,135 @@
+# PlanetarPy Charter
+
+PlanetaryPy is governed by the members of the Technical Committee (TC).
+
+## Statement of Purpose
+
+PlanetaryPy is a community effort to develop a core package
+for planetary science in Python and foster interoperability between
+Python planetary science packages.
+
+## Responsibilities
+
+The PlanetaryPy TC responsibilities include:
+
+* Technical direction
+* Setting release dates.
+* Release quality standards.
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+
+## Membership
+
+### Technical Committee Members
+
+For the current list of TC members, see the [member list](Members.md).
+
+
+TC seats are not time-limited.  There is no fixed size of the 
+TC.  However, the expected target is between 3 and 12, to ensure adequate
+coverage of important areas of expertise, balanced with the ability
+to make decisions efficiently.
+
+There is no specific set of requirements or qualifications for 
+TC membership beyond these rules.
+
+The TC may add additional members by consensus. If there
+are any objections to adding any individual member, an attempt
+should be made to resolve those objections following the 
+Consensus Seeking Process.
+
+A TC member may be removed from by voluntary resignation, or by
+consensus of all other members.
+
+Changes to TC membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full
+TC is not in attendance to participate, then the addition or removal
+is added to the agenda for the subsequent meeting.  This is to ensure
+that all members are given the opportunity to participate in all
+membership decisions.  If a TC member is unable to attend a meeting
+where a planned membership decision is being made, then their consent
+is assumed.
+
+No more than half of the TC members may be affiliated with the same
+employer.  If removal or resignation of a TC member, or a change of
+employment by a member, creates a situation where more than half of
+the membership shares an employer, then the situation must be
+immediately remedied by the resignation or removal of one or more 
+TC members affiliated with the over-represented employer(s).
+
+
+### Collaborators
+
+The PlanetaryPy GitHub repository is maintained by the 
+TC and additional Collaborators who are added on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access. These individuals
+are identified by the TC and their addition as Collaborators is
+discussed during the TC meeting.
+
```

Comment:
How about adding: Participation in the TC is **not** a requirement for becoming a Collaborator. Or is that self-understood by not mentioning it?

Reply:
Um, so the section this text is in is titled 'Collaborators' which is a sibling of a section (before it) called 'Technical Committee Members' and both are subsctions under Membership.  So I feel like the structure of the document pretty much makes it clear that these are two categories, and that they are different.

Reply:
well, Andrew seemed to have misunderstood it?

Reply:
Well, I think he was just making a general comment, I don't think he was reviewing the document during the meeting.